src/components/biggive-table/biggive-table.scss

```diff
@@ -0,0 +1,113 @@
+@include backgrounds();
+@include fills();
+
+:host {
+  display: contents;
+  @include standard-font();
+}
+
+
+.container {
+
+
+  $colour-primary-blue: #2C089B;
+$colour-secondary-green: #2AF135;
+$colour-tertiary-coral: #FF7272;
+
+$colour-brand-mhf-turquoise: #62CFC9; // Mental Health Fund
+$colour-brand-cc-red: #B30510; // Christmas Challenge
+$colour-brand-wgmf-purple: #6E0887; // Women and Girls Match Fund
+$colour-brand-gmf-green: #50B400; // Green Match Fund
+$colour-brand-emf-yellow: #FFE500; // Emergency Match Fund
+$colour-brand-c4c-orange: #F07D00; // Champions for Children
+$colour-brand-grey: #CBC8C8;
```

Comment:
Do we need to redefine all these colour vars here? It seems like it would make things a lot simpler if we could reference the colours from `src/globals/variables.scss` instead. I don't want to have to edit the table component when there's a new meta campaign with a new colour. And if I really have to edit it it would be much nicer if it's by adding a reference to something in the existing variables, not copying and pasting.

Reply:
Looking at this with @domtbg - she was saying the only colours we actually want to use in the table are the brand primary, secondary and tertiary colours. So could we avoid this entire discussion and make the component code much simpler by only supporting those three colours?

Reply:
These definitions were just left in there from me copying and pasting. They are not required themselves.

Reply:
I'm not sure I follow. Will you delete lines 17-23 then?

Reply:
Yes, they are not needed.